alacritty/src/window.rs

```diff
@@ -335,7 +350,10 @@ impl Window {
 
     #[cfg(not(any(target_os = "macos", windows)))]
     pub fn x11_window_id(&self) -> Option<usize> {
-        self.window().xlib_window().map(|xlib_window| xlib_window as usize)
+        #[cfg(feature = "x11")]
+        return self.window().xlib_window().map(|xlib_window| xlib_window as usize);
+        #[cfg(not(feature = "x11"))]
+        return None;
     }
```

Comment:
I don't think we need to define this for Wayland. Since we have a config in main.rs that just uses `None` for Windows/macOS anyways. So that should be consistent across cfgs here.

I'd be fine with just having a separate `x11_window_id` method for Windows/macOS/Wayland though if your goal is to clean up the callee a bit.

Reply:
Ah, I haven't noticed that, will try to move somewhere else.